CMakeLists.txt

```diff
@@ -94,6 +94,8 @@ target_include_directories(date_interface INTERFACE
 )
 
 
+install( TARGETS date_prj EXPORT DateConfig )
+install( EXPORT DateConfig DESTINATION lib/cmake/date )
```

Comment:
If this will be used on systems different from Linux, the destination must be set accordingly. ```find_package``` searches in different directories on different systems:
https://cmake.org/cmake/help/latest/command/find_package.html#command:find_package
An example to set it would be:
```
if(WIN32 AND NOT CYGWIN)
    set(DEF_INSTALL_CMAKE_DIR CMake)
else()
    set(DEF_INSTALL_CMAKE_DIR lib/cmake/SeqsiBuild)
endif()
```